fabric-content-registries-v1/src/main/java/net/fabricmc/fabric/mixin/content/registries/MixinFurnaceBlockEntity.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.content.registries;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+import net.minecraft.block.entity.FurnaceBlockEntity;
+import net.minecraft.item.ItemStack;
+
+import net.fabricmc.fabric.api.content.registry.v1.FuelRegistry;
+
+@Mixin(FurnaceBlockEntity.class)
+public class MixinFurnaceBlockEntity {
+	@Inject(at = @At("HEAD"), method="method_1144", cancellable = true)
+	private static void registerFuels(ItemStack stack, CallbackInfoReturnable<Integer> info){
+		FuelRegistry.getFuelMap().forEach((item, time)->{
```

Comment:
use the `get` method from the map

Reply:
why not use forEach?

Reply:
your forEach's time complexity is O(n)
get is O(1)

this is a big difference

Reply:
how will it work for all the items in the Map?

Reply:
It is a map right? the item is the key and the time is the value

Reply:
Yes it's a map and it will probably have multiple keys

Reply:
So the Item can be the key so you can get the value, which is the time, using the key

Reply:
Done.